velox/connectors/hive/storage_adapters/abfs/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# for generated headers
+
+if(VELOX_ENABLE_ABFS)
+  add_library(velox_abfs AbfsReadFile.cpp AbfsFileSystem.cpp AbfsReadFile.h
+                         AbfsFileSystem.h AbfsUtil.h)
+  target_link_libraries(velox_abfs Azure::azure-storage-blobs)
```

Comment:
You are missing several targets here (velox_file etc.). Please link all internal non-header targets and all external targets (e.g. Boost, Folly...) used in this target and don't rely on transient linking of targets. Also as mentioned below please use the scope keywords when linking targets.

Reply:
boost is header only, add other missed targets.